cmake/kokkos_compiler_id.cmake

```
@@ -94,6 +94,11 @@ if(Kokkos_ENABLE_HIP)
 
   string(REGEX MATCH "[0-9]+\\.[0-9]+\\.[0-9]+" TEMP_CXX_COMPILER_VERSION ${INTERNAL_COMPILER_VERSION_ONE_LINE})
   set(KOKKOS_CXX_COMPILER_VERSION ${TEMP_CXX_COMPILER_VERSION})
+
+  #FIXME HIP
```

```suggestion
  #FIXME_HIP
```
For easier grepping in the future along with other `FIXME_HIP`